playbooks/infra/reporting/roles/junit2json/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: Apache-2.0
+---
+# tasks file main.yml for role redhatci.ocp.junit2json
+
+# NOTE: in case somebody did not install the collection properly, we test for filter's dependency
+
+- name: Validate role variables
+  ansible.builtin.assert:
+    that:
+      - junit2_input_reports_list | length > 0
+      - junit2_output_dir | length > 0
+    fail_msg: "One of the mandatory variables is missing or empty"
+
+- name: Print input reports variable
+  ansible.builtin.debug:
+    var: junit2_input_reports_list
+
+- name: Initialize reports variable
+  ansible.builtin.set_fact:
+    junit2_reports_list: []
+    junit2_json_reports_list: []
+
+- name: Expand the input list to list of existing files
+  ansible.builtin.include_tasks:
+    file: expand.yml
+  loop: "{{ junit2_input_reports_list }}"
```

Comment:
this is not a list - in the default.yml it set as a string

Reply:
This variable is a list of str, defined here: `playbooks/infra/reporting/roles/junit2json/meta/argument_specs.yml`, the 1st variable.